src/status_im2/contexts/wallet/home/tabs/assets/view.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im2.contexts.wallet.home.tabs.assets.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im2.contexts.wallet.common.temp :as temp]
+    [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [tokens-loading? (rf/sub [:wallet/tokens-loading?])]
```

Comment:
`if-let` can be used here

Reply:
@vkjr - Yes, It's a good choice.

I used `let` as we will remove the `temp-token` list in the home soon and add a new subscription for this.